### PR TITLE
Fast calo maker

### PR DESCRIPTION
**Pull request by bechenard**
Fast CaloHit fix. This was done two weeks ago, I clicked on the "fetch upstream" button, and it will hopefully work.

### CODE REVIEW

**Comment by kutschke** [comment]
Does this address the issue that was originally found by Namitha and raised by Pasha at a Wednesday meeting a month or so ago?  Or is it something else?

**Comment by bechenard** [comment]
Yes, it does. I thought I asked Namitha to submit the PR while I was 
away, but I didn't, so I'm doing this now.

File: CaloReco/src/CaloHitMakerFast_module.cc [hunk 1]
```diff
@@ -39,10 +39,11 @@ namespace mu2e {
...
 	    fhicl::Atom<art::InputTag> pbttoken           { Name("ProtonBunchTimeTag"), Comment("ProtonBunchTime producer")};
             fhicl::Atom<double>        digiSampling       { Name("digiSampling"),       Comment("Digitization time sampling") }; 
-            fhicl::Atom<double>        deltaTPulses       { Name("deltaTPulses"),       Comment("Maximum time difference between two signals ") }; 
-            fhicl::Atom<double>        pulseRatioMin      { Name("pulseRatioMin"),      Comment("Min value of energy ratio between pulses ") }; 
-            fhicl::Atom<double>        pulseRatioMax      { Name("pulseRatioMax"),      Comment("Max value of energy ratio between pulses ") }; 
-            fhicl::Sequence<double>    caphriCrystalID    { Name("caphriCrystalID"),    Comment("List of caphri crystal ID") }; 
+            fhicl::Atom<double>        deltaTPulses       { Name("deltaTPulses"),       Comment("Maximum time difference between two signals") }; 
+            fhicl::Atom<double>        nPEperMeV          { Name("nPEperMeV"),          Comment("number of photo-electrons per MeV") };
```
Comment (brownd1978) [inline CaloReco/src/CaloHitMakerFast_module.cc:43]: Will any of these new parameters (noiseLevel, etc) be time dependent?  If so, I suggest a long-term plan to migrate them to conditions data, though that is beyond the scope of this PR.

**Review by brownd1978 — APPROVED** [review]

**Comment by bechenard** [comment]
This might be the case. We need to add support for conditions for the 
whole calo code, I'll do that for all calo related modules once Snowmass 
is done.

**Review by NamithaChitrazee — APPROVED** [review]

**Comment by NamithaChitrazee** [comment]
> @NamithaChitrazee I added you as a reviewer. Please check if this resolves the issues you brought up. It if does please approve with PR. Thanks.

Thank you. Yes, it does. I checked with these changes yesterday and I shall present a quick update on it in today's comp soft meeting.

**Comment by kutschke** [comment]
@FNALbuild run build test

**Pull request merged**